<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_action_classify_model_select</class>
 <widget class="QWidget" name="widget_action_classify_model_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select model</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General params</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QSlider" name="horizontalSliderTopK">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelTopK">
        <property name="text">
         <string>Top K</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelConfidence">
        <property name="text">
         <string>Confidence</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="horizontalSliderConfidence">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxConfidence">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxTopK">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSamplingRate">
        <property name="text">
         <string>Sampling rate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxSamplingRate">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="horizontalSliderSamplingRate">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSelectModel">
     <property name="title">
      <string>Select model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBoxSelectModel"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>283</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxConfidence</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderConfidence</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderConfidence</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxConfidence</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxTopK</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderTopK</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTopK</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxTopK</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSamplingRate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderSamplingRate</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSamplingRate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSamplingRate</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
